app/models/profile.rb

```diff
@@ -46,18 +46,22 @@ def pending_reasons
   end
 
   # rubocop:disable Rails/SkipsModelValidations
```

Comment:
Tangential to this PR since it was here already. Does this mean validations are enabled or disabled, and is that the way we want it?